### PR TITLE
Fix alignment issue in list content builder & peek height option

### PR DESCRIPTION
**Pull request by joyeeta26**
### Problem 
1. Alignment issue in List content builder
2. Peek height option not usable in demo app bottom sheet

### Root cause 
1. The issue was happening specifically when maxItemsInRow > itemsInRow and equidistant= false, because width ratio was being calculated as 1/itemsInRow.
2. Because peek height option was getting activated when bottom sheet gets activated, hence rendering it unusable 

### Fix
1. For euidistant false, the width ratio should be calculated as 1/maxItemsInRow.
2. Updated the condition so that peek height option is visible when bottom sheet is hidden

### Validations
for maxItemsInRow > size
image

For maxItemsInRow = size
image

For maxItemsInRow < size
image

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] Automated Tests
- [ ] Documentation and demo app examples
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and RTL layouts
- [ ] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

File: FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BottomSheetActivity.kt [hunk 1]
```diff
@@ -429,14 +429,14 @@ private fun CreateActivityUI() {
                     style = ButtonStyle.Button,
                     size = ButtonSize.Medium,
                     text = "+ 8 dp",
-                    enabled = !hidden,
+                    enabled = hidden,
```
Comment (Anubhvv) [inline FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BottomSheetActivity.kt:432]: not having context regarding this, can you tell me what is the reason for this change?

Comment (joyeeta26) [reply FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BottomSheetActivity.kt:432]: so currently in demo app, peek height option is only shown when drawer activates
image
and once the drawer activates, the option being unusable as it's hidden behind the scrim
so here we are making the change so that peek height option is enabled when the drawer is not activated so that we can configure the peek height before activating the drawer

**Review by Anubhvv — APPROVED** [review]

**Review by Dhoot — APPROVED** [review]
